components/resource_metering/src/recorder/sub_recorder/cpu.rs

```diff
@@ -47,6 +56,10 @@ impl SubRecorder for CpuRecorder {
         });
     }
 
+    fn collect(&mut self, records: &mut RawRecords, _: &mut HashMap<usize, LocalStorage>) {
+        records.keep_top_k(self.max_resource_groups);
```

Comment:
For per-second top K, we can leave only top 100 (MAX_MAX_RESOURCE_GROUPS = 100). BTW I think there is no need to do a 1-minute top K after that. What do you think? @zhongzc

Reply:
This may require two configuration items, one for per-second and one for per-minute(current `max_resource_groups`).

If we don’t need it to be configurable, we can simply use constant.

Reply:
We can drop the per-minute, and modifying the default value to be 100. In this way, no new config will be introduced.